src/styles/lists.styl

```diff
@@ -1,233 +1,187 @@
+        display block
+
+    .konnector-state
+        display none
+        position absolute
+        top 0
+        left 0
+        width 100%
+        height 100%
 
+    .item-wrapper
+        &:hover
+        &:focus
             .item-icon
-                width 4rem
-                height 4rem
-
-.item-wrapper
-    position         relative
-    box-sizing       border-box
-    border-radius    .25rem
-    margin           .53rem
-    padding-bottom   1rem
-    width 8.75rem
-    height 8.75rem
-    flex-shrink 0
-    cursor           pointer
-    text-decoration  none
-    text-align       center
-    transition       .2s ease all
-    &:hover
-        background-color white
-        transform  scale(1.05)
-        transition .2s ease all
+                box-shadow 0 rem(3) rem(8) 0 rgba(50, 54, 63, .15)
+                transform  scale(1.05)
+                transition .3s ease all
 
-    &.--add-service
+    .--add-service
         display flex
         flex-direction column
         justify-content center
 
-        .item-header .item-icon
-            height 2rem
-            width 2rem
-            border 2px dashed silver
-            border-radius 50%
-            padding calc(1rem - 2px) // consider 2px border
-            color coolGrey
+        &:hover
+        &:focus
+            .item-icon
+                box-shadow none
 
-        .item-add-service-label
+        .item-icon
+            box-sizing border-box
+            display flex
+            justify-content center
+            align-items center
+            padding rem(4)
+            background-color transparent
             color coolGrey
+            svg
+                width rem(56)
+                height rem(56)
+
+            @media (max-width: rem(466))
+                padding rem(2)
+
+                svg
+                    width rem(36)
+                    height rem(36)
+
+        .item-title
             text-transform uppercase
 
         &[aria-busy]
             opacity .8
 
             &:hover
-                background-color transparent
                 transform  none
 
-            .item-header
-                // Centers spinner
-                display flex
-                align-items center
-
-    .item-header
-        margin-bottom .5rem
-        border-radius 4px 4px 0 0
-        height        4.37em
-        display flex
-        justify-content center
-        align-items flex-end
-
-        .item-icon
-            width 3rem
-            height 3rem
-
-        .item-layered-item-icon
-            position relative
-            height 3rem
-            width 3rem
-
+.item-title
+    width 80%
+    margin rem(5) auto 0
+    color black
+    font-size .875rem
+    line-height 1.5
+    font-weight 700
+    white-space nowrap
+    overflow hidden
+    text-overflow ellipsis
+
+    @media (max-width: rem(466))
+
+        font-size rem(10)
+        line-height 1.8
+
+// Custom background override
+.has-custombg
```

Comment:
I think this class name does not match our naming conventions. It should be instead something like
`has-custom-bg`
or
`has-custom-background` (my preference, I f**king **LOVE** very explicit variable names)

Reply:
Fine by me :)